unattended_scripts/install_functions/opendistro/elasticsearch.sh

```diff
@@ -178,9 +178,9 @@ configureElasticsearch() {
     fi
 
     if [ -n "${single}" ]; then
-        copyCertificatesElasticsearch einame
+        copyCertificatesElasticsearch
     else
-        copyCertificatesElasticsearch einame pos
+        copyCertificatesElasticsearch
     fi
```

Comment:
This will make the call whether it is fulfilled or not

Reply:
Done in https://github.com/wazuh/wazuh-packages/commit/39c0d9730785973b0736a4b86c2994b9917ac73d